internal/controller/controllers/agent_controller.go

```diff
@@ -484,6 +487,31 @@ func (r *AgentReconciler) spokeKubeClient(ctx context.Context, clusterRef *aiv1b
 	return r.SpokeK8sClientFactory.CreateFromSecret(secret)
 }
 
+func (r *AgentReconciler) notifyNumberOfReboots(ctx context.Context, h *models.Host, node *corev1.Node, clusterRef *aiv1beta1.ClusterReference) error {
+	if !isNodeReady(node) {
+		return nil
+	}
+	secret, err := spokeKubeconfigSecret(ctx, r.Log, r.Client, r.APIReader, clusterRef)
+	if err != nil {
+		r.Log.WithError(err).Errorf("failed to get spoke secret for cluster %s/%s", clusterRef.Namespace, clusterRef.Name)
+		return err
+	}
+	kubeconfig, err := spoke_k8s_client.KubeconfigFromSecret(secret)
+	if err != nil {
+		r.Log.WithError(err).Errorf("failed to get kubeconfig from secret for cluster %s/%s", clusterRef.Namespace, clusterRef.Name)
+		return err
+	}
+	cmd := NewDebugCommands(r.Log, kubeconfig)
```

Comment:
why not to provide client?

Reply:
There was a request to use this as the image is already too large

Reply:
Now i understand, you actually run it as oc command